server.js

```diff
@@ -8,9 +8,12 @@ require('dotenv').config()
 
 // Set up Sentry reporting as early in the process as possible.
 const config = require('config').util.toObject()
-const Raven = require('raven')
-Raven.config(process.env.SENTRY_DSN || config.private.sentry_dsn).install()
-Raven.disableConsoleAlerts()
+const Sentry = require('@sentry/node')
+Sentry.init({
+  dsn: process.env.SENTRY_DSN || config.private.sentry_dsn,
+  integrations: integrations =>
+    integrations.filter(integration => integration.name !== 'Console'),
```

Comment:
This removes console log output which seems to be unrelated to the request that caused the error.